### PR TITLE
Rails 4 2

### PR DESCRIPTION
**Pull request by yurifrl**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://azclick-rails-4-2.spree-spree-9ed724b77adec04c49c2.ttrcloud.com' target='_blank'>Click to open.</a>

File: core/app/models/spree/stock/coordinator.rb [hunk 1]
```diff
@@ -10,7 +10,10 @@ def initialize(order, inventory_units = nil)
 
       def shipments
         packages.map do |package|
-          package.to_shipment.tap { |s| s.address = order.ship_address }
+          package.to_shipment.tap do |s|
+            s.address = order.ship_address
+            s.state = :pending
```
Comment (JDutil) [inline core/app/models/spree/stock/coordinator.rb]: Why is this necessary?

**Comment by JDutil** [comment]
I don't know what you're doing.
